README.md

```diff
@@ -1,18 +1,129 @@
-extensible-effects is based on the work
-[Extensible Effects: An Alternative to Monad Transformers](http://okmij.org/ftp/Haskell/extensible/).
-Please read the [paper](http://okmij.org/ftp/Haskell/extensible/exteff.pdf) and
-the followup [freer paper](http://okmij.org/ftp/Haskell/extensible/more.pdf) for
-details. Additional explanation behind the approach can be found on [Oleg's website](http://okmij.org/ftp/Haskell/extensible/).
+
+# Extensible effects
 
 [![Build Status](https://travis-ci.org/suhailshergill/extensible-effects.svg?branch=master)](https://travis-ci.org/suhailshergill/extensible-effects)
 [![Join the chat at https://gitter.im/suhailshergill/extensible-effects](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/suhailshergill/extensible-effects?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Stories in Ready](https://badge.waffle.io/suhailshergill/extensible-effects.png?label=ready&title=Ready)](http://waffle.io/suhailshergill/extensible-effects)
 [![Stories in progress](https://badge.waffle.io/suhailshergill/extensible-effects.png?label=in%20progress&title=In%20progress)](http://waffle.io/suhailshergill/extensible-effects)
 
-## Advantages
+*Implement effectful computation in a modular way!*
+
+The main and only monad is built upon `Eff` from `Control.Eff`. `Eff r a` is parameterized by
+the effect-list `r` and the monadic-result type `a` similar to other monads.
+
+In case you know monad transformers or mtl: This library provides only one monad that includes all your effects instead of layering different transformers. It is not necessary to lift the computations through a monad stack. Also, it is not required to lift every `Monad*` (like `MonadError`) typeclass though all transformers.
```

Comment:
I intenionally did not while working on the text. Since changes to the text leads to re-indentation. I will indent it correctly for the final version.

Reply:
the editor doesn't do that automatically? fyi, you can configure emacs/vim to do that as you type.